controllers/discovery/initDiscoveryDomainData.js

```diff
@@ -24,10 +26,12 @@ const InitDiscoveryDomainData = async (req, res) => {
                 AND A.user_id_follower = :userId
             RIGHT JOIN domain_page C 
                 ON C.domain_page_id = A.domain_id_followed
+            WHERE C.credder_score >= ${CREDDER_MIN_SCORE}
             GROUP BY A.domain_id_followed, domain_page_id, A.user_id_follower, C.domain_name, C.logo, C.short_description, C.credder_score
             ORDER BY 
                 common DESC, 
-                A.domain_id_followed ASC
+                A.domain_id_followed ASC,
+                C.credder_score DESC
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the WHERE clause to filter by `credder_score` is a good way to ensure only relevant results are returned. However, directly injecting the `CREDDER_MIN_SCORE` into the SQL query can potentially lead to SQL injection if the value of `CREDDER_MIN_SCORE` is somehow compromised or manipulated. It would be safer to use parameterized queries or prepared statements instead.
```diff
- WHERE C.credder_score >= ${CREDDER_MIN_SCORE}
+ WHERE C.credder_score >= :credderMinScore
```
And then in your replacements object, you would add:
```javascript
replacements: {
    userId: userId,
    limit: limit,
    offset: offset,
    credderMinScore: CREDDER_MIN_SCORE
}
```
This ensures that the value is properly escaped and safe from SQL injection attacks.


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
this is defined as constant, so will not lead to SQL Injection